Frameworks/Foundation/NSDictionary.mm

```diff
@@ -716,6 +716,20 @@ - (NSArray*)keysSortedByValueUsingComparator:(NSComparator)comparator {
 /**
  @Status Interoperable
 */
+- (NSArray*)keysSortedByValueWithOptions:(NSSortOptions)opts usingComparator:(NSComparator)cmptr {
+    NSMutableArray* ret = [NSMutableArray arrayWithArray:[self allKeys]];
```

Comment:
or autorelease it

---
In reply to: [72358602](https://github.com/Microsoft/WinObjC/pull/673#discussion_r72358602) [](ancestors = 72358602,71627991)